tests/DCPS/Messenger_Annotation/Messenger.idl

```diff
@@ -5,18 +5,23 @@
  * See: http://www.opendds.org/license.html
  */
 
+@default_nested(FALSE)
 module Messenger {
 
   typedef char hexadecimal_t[8];
 
+  @nested
+  struct Hidden {
+    string hidden_message;
```

Comment:
You don't have to change this, but I just want to make sure you understand this isn't actually hidden in any way.